modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java

```diff
@@ -62,14 +62,14 @@ public void testSessionExpiration() throws Exception {
         sql("CREATE TABLE TST(id INTEGER PRIMARY KEY, val INTEGER)");
         sql("INSERT INTO TST VALUES (1,1), (2,2), (3,3), (4,4)");
 
-        Session ses1 = sql.sessionBuilder().defaultPageSize(1).idleTimeout(1, TimeUnit.MILLISECONDS).build();
+        Session ses1 = sql.sessionBuilder().defaultPageSize(1).idleTimeout(1, TimeUnit.SECONDS).build();
```

Comment:
I found : public static final long SESSION_EXPIRE_CHECK_PERIOD = TimeUnit.SECONDS.toMillis(1);
Why do we need hardcoded: **long timeout** ? Also comment : "SessionManager.checkPeriod" confusing.